client/ayon_usd/plugins/publish/integrate_usd_output_processor_remap_relative.py

```
@@ -0,0 +1,133 @@
+import os
+
+import pyblish.api
+
+from ayon_core.pipeline import OptionalPyblishPluginMixin
+from ayon_core.pipeline.publish.lib import get_instance_expected_output_path
+
+# Avoid USD imports turning into errors when running in a host that does
+# not support the USD libs.
+try:
+    from pxr import Sdf, UsdUtils
+    HAS_USD_LIBS = True
+except ImportError:
+    HAS_USD_LIBS = False
```

whats the point in checking this in the end we need UsdUtils because we use it in a function at the end ? 

i mean yes you have a log in the plugin but i think an error would be a bit better regarding the fail fast principle.

---

This plug-in will always also be loaded in DCCs that do not have the USD libs, e.g. TVPaint if the USD addon happens to be enabled in the bundle. In that case we do not want to clutter the logs with errors about the USD addon.

It's somewhat similar to what's done here https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/plugins/publish/extract_usd_layer_contributions.py#L6-L10

---

yeah i see what you mean i think in terms of implementation its not the best but if we have no way of using something like an early return to solve it this might be the best we can doo. 

i will try and test if some early return would work.